lib/common/http1client.c

```diff
@@ -60,6 +60,7 @@ struct st_h2o_http1client_t {
     char _chunk_len_str[(sizeof(H2O_UINT64_LONGEST_HEX_STR) - 1) + 2 + 1]; /* SIZE_MAX in hex + CRLF + '\0' */
     h2o_buffer_t *_body_buf;
     h2o_buffer_t *_body_buf_in_flight;
+    uint64_t _last_bytes_read;
```

Comment:
Am I correct in assuming that the role of this variable is to maintain the number of bytes being already processed on the associated socket?

Assuming that is the case, I think it might be better to: 1) clarify that as a doc-comment 2) change the name to something like "_socket_bytes_processed".

"last_bytes_read" is confusing as the value does not indicate that.

Reply:
done in https://github.com/h2o/h2o/pull/2240/commits/0eb8ffeba6d9f8cc1c80fcca5b5e9e60d6649e5c